### PR TITLE
Adding BICO token on Kovan and Mainnet

### PR DESCRIPTION
**Pull request by tomarsachin2271**
**Description**
Adding Biconomy Token to Optimism network (Both Test and Mainnet). 

**Additional context**
To be able to bridge BICO token from ethereum kovan/Mainnet to optimism kovan/Mainnet using app.optimism.io. 

**Metadata**

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1525,6 +1525,17 @@
...
+    {
+      "chainId": 42,
+      "address": "0xEf2E078a649aAd3908b7F4f9aD75A881D4f3b7e3",
+      "name": "Biconomy Token",
+      "symbol": "BICO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BICO.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
```
Comment (Nicca42) [inline optimism.tokenlist.json:1538]: Please add the information for the token on Optimism Kovan. Thanks!

**Review by Nicca42 — CHANGES_REQUESTED** [review]

File: optimism.tokenlist.json (hunk 1 above)
Comment (tomarsachin2271) [reply optimism.tokenlist.json:1538]: Done

**Comment by tomarsachin2271** [comment]
@imranjami Please check and confirm this PR. This is to add BICO token to the optimism test network.

**Comment by tomarsachin2271** [comment]
@Nicca42 Please check the PR. It has been updated as requested.

**Comment by tomarsachin2271** [comment]
@Nicca42 Can you please check this PR

File: optimism.tokenlist.json [hunk 2]
```diff
@@ -1569,6 +1569,50 @@
...
+    },
+    {
+      "chainId": 1,
+      "address": "0xf17e65822b568b3903685a7c9f496cf7656cc6c2",
+      "name": "Biconomy Token",
+      "symbol": "BICO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BICO.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
```
Comment (Nicca42) [inline optimism.tokenlist.json:1614]: The [address of the bridge on L1 is 0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1](https://etherscan.io/address/0x99C9fc46f92E8a1c0deC1b1747d010903E884bE1#code). Please update this.

File: optimism.tokenlist.json [hunk 3]
```diff
@@ -1569,6 +1569,50 @@
...
+    },
+    {
+      "chainId": 42,
+      "address": "0xEf2E078a649aAd3908b7F4f9aD75A881D4f3b7e3",
+      "name": "Biconomy Token",
+      "symbol": "BICO",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/BICO.png",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
```
Comment (Nicca42) [inline optimism.tokenlist.json:1581]: The [address of the bridge on L1 is 0x22F24361D548e5FaAfb36d1437839f080363982B](https://kovan.etherscan.io/address/0x22F24361D548e5FaAfb36d1437839f080363982B#code). Please update this.

**Review by Nicca42 — CHANGES_REQUESTED** [review]

**Comment by tomarsachin2271** [comment]
Update the bridge address as per your comments. Please check @Nicca42

**Comment by smartcontracts** [comment]
Hi! We recently updated our token list to a new v2 system that simplifies the process of adding new tokens and will ultimately speed up the review process. Please refer to our [new README](https://github.com/ethereum-optimism/ethereum-optimism.github.io#adding-a-token-to-the-list) for information about the new process. You may have to close this PR and open a new one instead. Apologies for the inconvenience.

**Comment by smartcontracts** [comment]
Hi! We recently updated our token list to a new v2 system that simplifies the process of adding new tokens and will ultimately speed up the review process. Please refer to our [new README](https://github.com/ethereum-optimism/ethereum-optimism.github.io#adding-a-token-to-the-list) for information about the new process.

Our new token listing process is significantly simplified and makes the steps to add a new token much easier. Since this PR is using the old token listing format, we are closing this PR and would like to ask you to open a new PR with the updated format. Our apologies for the inconvenience here, we're doing our best with the new process to make sure that future token listing PRs get feedback quickly.

**Pull request closed without merging**